docs/documents/powerbi.md

```diff
@@ -0,0 +1,25 @@
+The powerBI db backup pipeline was migrated from the old paas environment to the new aks environment
+
+the old pipeline was implemented using azure devops classic release pipeline. As at the time of writing this document the old pipeline was located here
+https://dev.azure.com/dfe-ssp/S105-School-Experience/_release?_a=releases&view=mine&definitionId=64
+
+The new pipeline is also implemented using the classic azure devops release pipeline and it is located here
+https://dev.azure.com/dfe-ssp/S105-School-Experience/_release?_a=releases&view=mine&definitionId=66
+
+the pipeline works by doing a backup of the kubernetes application database in aks. The database is accessed byport forwarding using Konduit.
+
+konduit is installed using the make file from the repository
+
+a dump command is then run on the postgres database to dump the db into a file on the pipeline
+the command that is run is
+``bin/konduit.sh  -k $(KV) -d gse-production get-school-experience-production  -- pg_dump  -E utf8 --clean --if-exists --no-owner --verbose --no-password  -f  schoolexperiencedb_temp.sql``
+where kv is the keyvault name, -d option refers to the database name, and get-school-experience-production  refers to the pod name
+
+the performance db is then deleted and recreated in the next task and the exported db in schoolexperiencedb_temp.sql  is then imported into the performance database
+
+ogr2ogr is also used to load GeoJSON files (eer.json and topo_lad.json) into the PostgreSQL database
+
+at the PowerBI end, in order to change the database credentials and(possibly the reference db in future if necessary) we carried out the following steps
+go to Settings -> Manage connections and gateways
+open the data source options menu and choose settings
+change  username and password
```

Comment:
Please mention there are 2 sets of credentials: admin and perf user, how they are set up and which one is used in the datasource